lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -666,25 +665,25 @@ class EngineParagraphStyle implements ui.ParagraphStyle {
   }
 
   @override
-  bool operator ==(dynamic other) {
+  bool operator ==(Object other) {
     if (identical(this, other)) {
       return true;
     }
     if (other.runtimeType != runtimeType) {
       return false;
     }
-    final EngineParagraphStyle typedOther = other;
-    return _textAlign == typedOther._textAlign ||
-        _textDirection == typedOther._textDirection ||
-        _fontWeight == typedOther._fontWeight ||
-        _fontStyle == typedOther._fontStyle ||
-        _maxLines == typedOther._maxLines ||
-        _fontFamily == typedOther._fontFamily ||
-        _fontSize == typedOther._fontSize ||
-        _height == typedOther._height ||
-        _textHeightBehavior == typedOther._textHeightBehavior ||
-        _ellipsis == typedOther._ellipsis ||
-        _locale == typedOther._locale;
```

Comment:
The old version looked quiet strange with `||` instead of `&&`.

Reply:
This had to be a bug.